deployment/README.md

```diff
@@ -74,4 +75,26 @@ $ export DOCKER_CERT_PATH=/path/to/docker/certs
 # Set application version
 $ export GIT_COMMIT=1234567
 $ ./scripts/load_configured_data --production
-```
\ No newline at end of file
+```
+
+### Fixing Staging
+Depending on the configuration changes, staging will sometimes get into a bad state after deployment as we do not handle all types of config changes perfectly. As a result, you may need to "wipe the slate clean" in staging to see your changes. You can do this by ssh-ing into the staging app server, downloading the appropriate script, and running it, like so (**NOTE: the database will be dropped and recreated!**):
+
+```bash
+$ ssh-add ~/.ssh/district-builder.pem
+...
+$ ssh -A -i ~/.ssh/district-builder.pem ec2-user@34.207.181.197
+...
+$ ssh ec2-user@10.0.0.237
+...
+# Remove existing copy of script (if it exists)
```

Comment:
Prior to this step I had to execute `docker exec -ti districtbuilder-django bash`.

Reply:
Whoops! Good catch.